### DISCUSSION TITLE
Easy way to use your own custom license.json file

### DISCUSSION THREAD

**Discussion by bluthen** [sandworm-hq/sandworm-audit#93] [Ideas]
I think it can be useful to have an option to specify your own licenses.json. With your own licenses.json you can put whatever string of license you want in it. You can have your own categories.
```
"licenses_db": ".sandworm.licenses.json"
```

Related to #90 

Example, like CC-SA licenses are currently in Uncategorized (which should still bring up an issue for you to resolve), but I would probably put it in Protective. I can see where you might want your own categories of licenses for your particular project: non-commercial, not-gpl-compatible, etc.

**Comment by gabidobo** [comment]
Thank @bluthen!

Yeah being able to have your own categories makes sense.

Maybe something like this, in `.sandworm.config.json`?

```json
{
  "audit": {
    "licensePolicy": {
      "categories": [
        { "name": "Custom", "licenses": ["MIT", "Apache-2.0"] }
      ],
      "high": ["cat:Network Protective", "cat:Strongly Protective"],
      "low": ["cat: Custom"],
    }
  }
}
```

**Reply by bluthen** [reply]
Yeah, I think that would be great!

I was looking for a good table comparison for anyone wanting to make your own categories. I know it is just a wikipedia page, so you need to check it, but I did find this:

https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licenses

SPDX list:
https://spdx.org/licenses/

**Comment by gabidobo** [comment]
Done in #99 and released in v1.36.0!

Updates to the docs are incoming, some notes are available in the PR description for now.
